src/CalendarUtils.php

```diff
@@ -670,6 +670,11 @@ private static function getMonthNames($month, $shorten = false, $len = 3)
         return ($shorten) ? mb_substr($ret, 0, $len, 'UTF-8') : $ret;
     }
 
+    public static function getMonthName($month, $shorten = false, $len = 3)
```

Comment:
I mean, define new public and human-readable methods like these:
- `getMonthName(int $month): string`
- `getMonthShortName(int $month): string`
- `getMonthShortNameBy(int $month, int $lentgh): string`

and please write new tests for your defined methods